modules/websocket/websocket_server.cpp

```diff
@@ -67,6 +71,15 @@ void WebSocketServer::_bind_methods() {
 	ADD_SIGNAL(MethodInfo("data_received", PropertyInfo(Variant::INT, "id")));
 }
 
+String WebSocketServer::get_bind_ip() const {
+	return bind_ip;
+}
+
+void WebSocketServer::set_bind_ip(const String &p_bind_ip) {
+	ERR_FAIL_COND(is_listening());
```

Comment:
Should also check the validity of the IP address:
`ERR_FAIL_COND(!p_bind_address.is_valid() && !p_bind_address.is_wildcard())`

Reply:
Why disallowing setting a wildcard?

Reply:
added